code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -25,6 +25,7 @@
 	color = "#808080" // rgb: 128, 128, 128
 	taste_description = "oil"
 	evaporation_rate = 2
+	condensating_point = 1000 //fuck you this goes away
```

Comment:
Change to a define so the reasoning is understood. "INSTANT_CONDENSATION"
Remove the comment